rebar.config

```diff
@@ -67,7 +67,7 @@
         {aeminer, {git, "https://github.com/aeternity/aeminer.git",
                    {ref, "0a82f0f"}}},
 
-        {aebytecode, {git, "https://github.com/aeternity/aebytecode.git", {ref, "135a27c"}}},
+        {aebytecode, {git, "https://github.com/aeternity/aebytecode.git", {branch, "factories"}}},
```

Comment:
but proper hash when available or if you really want use 'master'

Reply:
I will fix it when aebytecode gets merged